### DISCUSSION TITLE
BUG: Cannot respond to the first picture

### DISCUSSION THREAD

**Discussion by SaltedFish23** [open-webui/open-webui#6153] [General]
# Bug Report

## Installation Method

Docker

## Environment

- **Open WebUI Version:** v0.3.32

- **Operating System:** Windows11, WSL2

**Confirmation:**

- [ ] I have read and followed all the instructions provided in the README.md.
- [ ] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [ ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

gpt can see the picture and respond to my prompt.

## Actual Behavior:

Gpt cannot respond to the prompt. It seems that it cannot see the picture. But in the next run, it can respond correctly.
![image](https://github.com/user-attachments/assets/bf6ea140-947e-49b5-99f8-2d9e8d2a02ce)

## Description

**Bug Summary:**
Chat cannot see the picture at first, it can only see it after the second question.

## Reproduction Details

**Steps to Reproduce:**
Upload a picture with question.

## Logs and Screenshots

**Browser Console Logs:**
None

**Docker Container Logs:**
None

**Screenshots/Screen Recordings (if applicable):**
See `Actual Behavior`

## Additional Information

None

**Comment by tjbck** [comment]
Unable to reproduce, keep us updated!

![image](https://github.com/user-attachments/assets/aa310283-a51b-4c40-a8f9-94cd958f393d)

**Reply by SaltedFish23** [reply]
Have you try to add pipelines to `Connections`?
I find that if i remove the pipelines from `OpenAI API`,
![image](https://github.com/user-attachments/assets/9956cc77-7dc5-4f1b-9371-1a508141d546)

everything goes well:
![image](https://github.com/user-attachments/assets/a353d78b-e45f-4d05-b565-cdf40477b5f8)

But if a add pipelines to API:
![image](https://github.com/user-attachments/assets/8c084c0e-c684-4822-bb2d-bc04bbef1281)

the bug occurs:
![image](https://github.com/user-attachments/assets/8bee8291-a1eb-48d3-a76d-1d82cc4d02c4)
